templates/zerver/api/fetch-api-key.md

```diff
@@ -0,0 +1,34 @@
+# Fetch a production API key
+
+{generate_api_description(/fetch_api_key:post)}
+
+## Usage examples
+
+{start_tabs}
+{tab|curl}
+
+{generate_code_example(curl)|/fetch_api_key:post|example}
+
+{end_tabs}
+
+## Parameters
+
+{generate_api_arguments_table|zulip.yaml|/fetch_api_key:post}
+
+## Response
+
+### Return values
+
+{generate_return_values_table|zulip.yaml|/fetch_api_key:post}
+
+### Example response
+
+A typical successful JSON response may look like:
+
+{generate_code_example|/fetch_api_key:post|fixture(200)}
+
+This endpoint needs password, if you signed up using passwordless
+authentication and never had a password, you can [reset your password](/help/change-your-password).
```

Comment:
@shanukun Thanks for working on this! Looks great! Can you move this line just above `usage examples` in

> 
> ```
> !!! warn ""
>     **Note:** text
> ```

type of block. I think many will miss reading it down there.

Reply:
I edited it. Can you please have a look at it. Thank you.

Reply:
Looks good! @timabbott will do the final review when he gets time.